### ISSUE TITLE
xml file in UTF-8 generated from SQL Server bcp: content shown only as <?><?>...

### ISSUE THREAD

**Issue by aisbergde** [microsoft/vscode#98144]
Issue Type: <b>Bug</b>

I use SQL Server 2017 to create XML files on windows 10, for example:

`bcp "select [XML_PK_or_Relation] from [aaa].[SchemaMeta].[XML_diverse] WHERE [Schema_Name] = 'bbb'" queryout D:\bbb.meta.xml -w -T`

the souce column has XML datatype

The exported files are shown as required in notepad++, information about the file is: 
Windows(CR LF); UCS-2 LE BOM

When I open the generated files in VS Code I see only questionmarks in a square, like
`<?><?>`

I append a file.

VS Code version: Code 1.45.1 (5763d909d5f12fe19f215cbfdd29a91c0fa9208a, 2020-05-14T08:27:35.169Z)
OS version: Windows_NT x64 10.0.18363

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7500U CPU @ 2.70GHz (4 x 2904)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.87GB (12.11GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (42)</summary>

Extension|Author (truncated)|Version
---|---|---
excel-to-markdown-table|csh|1.2.2
vscode-markdownlint|Dav|0.35.2
xml|Dot|2.5.0
prettier-vscode|esb|4.7.0
vscode-pull-request-github|Git|0.16.0
azure-pipelines|ms-|1.165.1
vscode-azureappservice|ms-|0.16.5
vscode-azurefunctions|ms-|0.22.1
vscode-azureresourcegroups|ms-|0.1.0
vscode-azurestorage|ms-|0.9.0
vscode-azurevirtualmachines|ms-|0.1.0
vscode-cosmosdb|ms-|0.12.2
vscode-docker|ms-|1.2.0
csharp|ms-|1.21.18
vscode-dotnet-runtime|ms-|0.1.2
mssql|ms-|1.9.0
python|ms-|2020.5.78807
remote-containers|ms-|0.117.1
remote-ssh|ms-|0.51.0
remote-ssh-edit|ms-|0.51.0
remote-wsl|ms-|0.44.2
vscode-remote-extensionpack|ms-|0.20.0
azure-account|ms-|0.8.11
azurecli|ms-|0.5.0
vscode-node-azure-pack|ms-|0.0.11
azurerm-vscode-tools|msa|0.10.0
java|red|0.61.0
jstool|sun|1.2001.1
ghosttext|tok|1.1.0
vscodeintellicode|Vis|1.2.7
azure-iot-edge|vsc|1.21.0
azure-iot-toolkit|vsc|2.16.1
azure-iot-tools|vsc|0.3.0
vscode-iot-device-cube|vsc|0.2.0
vscode-iot-workbench|vsc|0.14.0
vscode-java-debug|vsc|0.26.0
vscode-java-dependency|vsc|0.10.1
vscode-java-pack|vsc|0.9.1
vscode-java-test|vsc|0.22.4
vscode-maven|vsc|0.21.4
markdown-pdf|yza|1.4.4
markdown-all-in-one|yzh|2.8.0

</details>

[sti.meta.xml.zip](https://github.com/microsoft/vscode/files/4649018/sti.meta.xml.zip)

**Comment by kieferrm** [comment]
Does that also happen when you disable the XML extension?

**Comment by aisbergde** [comment]
I disabled "Prettier - Code formatter" and "XML Tools", restarted, and the formatting was OK.
I enabled "XML Tools", restarted, and it still works fine.
I enabled "Prettier - Code formatter", restarted, and it still works fine.

Currently I can't reproduce the bug. But I got it a while otherwise I would not open the issue.

Maybe the order of plugins when they have been enabled is relevant. No idea.
